.gitignore

```diff
@@ -2,7 +2,8 @@
 # pipenv environment
 Pipfile*
 .envrc
-.vocode
+.vscode
```

Comment:
Could you use `.git/info/exclude` instead `.gitignore` to ignore your private file?
https://medium.com/@dave_lunny/exclude-files-from-git-without-committing-changes-to-gitignore-986fa712e78d
Or Try `$HOME/.config/git/ignore` for your global setting.